src/pages/resultsView/survival/SurvivalUtil.tsx

```diff
@@ -59,11 +59,11 @@ export const survivalPlotTooltipxLabelWithEvent: {
     DSS: 'Time of Death',
 };
 
-export const plotsPriority: { [prefix: string]: number } = {
-    OS: 1,
-    DFS: 2,
-    PFS: 3,
-    DSS: 4,
+export const RESERVED_SURVIVAL_PLOT_PRIORITY: { [prefix: string]: number } = {
```

Comment:
need a comment explaining concept of RESERVED

Reply:
Sure